.github/workflows/welcome-new-contributors.yml

```diff
@@ -19,11 +19,11 @@ jobs:
 
             - Make sure you've read [this self-check checklist](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide#self-check).
             - Translations that have not been reviewed by a native speaker will be merged after a minimum of 1 week. If there are no reviewers for your language, please mention it explicitly.
-            - Refrain from continually updating the branch with `master` yourself. A maintainer will do this before merging anyway.
+            - Refrain from continually updating the branch with `master` yourself. A [maintainer](https://osu.ppy.sh/wiki/en/People/osu%21_wiki_maintainers) will do this before merging anyway.
             - Apply review suggestions in batches from the `Files changed` tab like this:
 
               ![Demo GIF](https://user-images.githubusercontent.com/36758269/214177263-b6ead137-e7e0-43a0-9aec-00f81894ebf7.gif)
 
             See the [contribution guide](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide) for more information.
 
-            Because this is your first time contributing, running [CI checks](https://osu.ppy.sh/wiki/en/osu%21_wiki/Maintenance#ci-checks) (common error detection) requires approval from a [maintainer](https://osu.ppy.sh/wiki/en/People/osu%21_wiki_maintainers). If you need help with decrypting CI check error messages, or with any other issues, ask in the `#osu-wiki` channel on the [osu!dev Discord server](https://discord.com/invite/ppy).
+            [CI checks](https://osu.ppy.sh/wiki/en/osu%21_wiki/Maintenance#ci-checks) provide automatic detection for common errors. If you need help with decrypting CI check error messages, or with any other issues, ask in the `#osu-wiki` channel on the [osu!dev Discord server](https://discord.com/invite/ppy).
```

Comment:
the change puts this paragraph in an odd position because it feels disconnected from the previous body of text, maybe add a one-two-three word transition to what these checks are (even the previous version was more smooth in this regard)?

Reply:
like this?

```suggestion
            All changes are run through [continuous integration checks](https://osu.ppy.sh/wiki/en/osu%21_wiki/Maintenance#ci-checks), which provide automatic detection for common errors. If you need help with decrypting CI check error messages, or with any other issues, ask in the `#osu-wiki` channel on the [osu!dev Discord server](https://discord.com/invite/ppy).
```

Reply:
yes this is good enough